docs/curators/CURATOR_TOOLS_BASTION.md

```diff
@@ -0,0 +1,249 @@
+
+On the dataset admin page, you will be able to create a mockup page in order to preview the final dataset view page with the information that was added to the database in the previous steps.
+
+## 6. `transfer` - copy dataset files into Wasabi
+
+When all files in a dataset have been finalised and curated then they can be copied into Wasabi using the `transfer` tool. The path to the user dropbox directory is provided as the `--sourcePath` parameter with the value of the `--doi` parameter being the DOI for the dataset. The `--wasabi` flag inform the `transfer` tool to copy files into Wasabi storage. The `--apply` flag takes the `transfer` tool out of dry-run mode that results in the actual transfer of files into Wasabi storage from the user drop box directory.
+```
+[peterl@ip-10-99-0-95 user5]$ transfer --doi 102498 --sourcePath /share/dropbox/user5/ --wasabi --apply
+```
+
+## 7. Housekeeping of user dropboxes of published datasets
+
+After dataset files have been copied into Wasabi, the files should also be backed up into S3 Glacier:
+```
+[peterl@ip-10-99-0-95 user5]$ transfer --doi 102498 --sourcePath /share/dropbox/user5/ --backup --apply
+```
+
+After you have confirmed the files are safely stored in Wasabi and Glacier then the `user` and `user.orig` dropbox directories should be deleted to save storage space after the manuscript has been published.
+
+## `postUpload`: a wrapper script to create readme file and update file metadata in database
+
+> [!IMPORTANT]
+> To execute the `postUpload` command, change directory to the dataset's associated user dropbox directory located at `/share/dropbox/`:
+```
+[peterl@ip-10-99-0-95 ~]$ cd /share/dropbox/user5
+```
+
+There is a script called `postUpload` which calls `createReadme`, `calculateChecksumSizes` and `fileMetaToDb` in turn so that these three tools do not have to be manually executed one after another:
+```
+# Ensure you are in the dropbox directory
+[peterl@ip-10-99-0-88 ~]$ pwd
+/share/dropbox/user5
+[peterl@ip-10-99-0-88 ~]$ sudo postUpload --doi 102498 --dropbox user5
+Creating README file for 102498
+[DOI]
+10.5524/102498
+...
+[Comments]
+
+[End]
+Created readme file and uploaded it to Wasabi gigadb-website/staging bucket directory
+Creating dataset metadata files for 102498
+Created 102498.md5
+Created 102498.filesizes
+Updating file sizes and MD5 values in database for 102498
+Updating md5 checksum values as file attributes for 102498
+Number of changes: 3
+Updating file sizes for 102498
+Number of changes: 3
+Updated file metadata for 102498 in database
+```
+> [!TIP]
+> Take note of the number of changes made by the md5 and file size update tool. This number should be equal to the number of files listed in the metadata files.
+
+To ensure the postUpload script has worked, you should perform checks using the dataset, sample and file admin pages to see if dataset metadata are correctly stored in the database.
```

Comment:
I dont think any of the postUpload scripts effect the sample table in any way.

Reply:
I have removed mention of samples here now.